user.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Try removing every bootstrap class and id which is not needed in your html code.